nix/ec2-rds-subnet-group.nix

```diff
@@ -0,0 +1,38 @@
+{ config, lib, uuid, name, ... }:
+
+with lib;
+with import ./lib.nix lib;
+
+{
+  options = {
+    name = mkOption {
+      default = "nixops-${uuid}-${name}";
+      type = types.str;
+      description = ''
+        Name of the RDS DB subnet group.
+      '';
+    };
+
+    description = mkOption {
+      type = types.str;
+      description = ''
+        Description of the RDS DB subnet group.
+      ''; 
+    };
+
+    imports = [ ./common-ec2-options.nix ];
```

Comment:
This addition wasn't tested, can you please make sure that it works ?
It doesn't evaluate as ./common-ec2-options.nix is a function so please make it's merged to options
```
options = {
   /* options declarations */
} // (import ./common-ec2-options.nix { inherit lib; });
```
Also if I fix that locally the diff engine will throw an error as it doesn't find a handler that realizes the `tags` diff, so it needs to be fixed in the python side to create a handler for the `tags` option.